addons/account_payment/views/account_portal_templates.xml

```diff
@@ -1,12 +1,96 @@
 <odoo>
+    <template id="portal_my_home_overdue_invoice" name="Portal layout: overdue invoices" inherit_id="portal.portal_breadcrumbs" priority="30">
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
+            <li t-if="page_name == 'overdue_invoices'" t-attf-class="breadcrumb-item active">
+                <a t-attf-href="/my/invoices?{{ keep_query() }}">Invoices &amp; Bills</a>
+            </li>
+            <li t-if="payment" class="breadcrumb-item active">
+                <t t-out="payment['reference']"/>
+            </li>
+        </xpath>
+
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="after">
+            <a t-if="page_name == 'invoice' and not invoice and filterby != 'bills' and overdue_invoice_count > 0"
+                class='btn btn-primary'
+                href='/my/invoices/overdue'>
+                Pay overdue
+            </a>
+        </xpath>
+    </template>
+
+    <template id="account_payment.portal_docs_entry" inherit_id="portal.portal_docs_entry">
+        <xpath expr="//a" position="inside">
+            <t t-out="html_children"/>
+        </xpath>
+    </template>
+
+    <template id="portal_my_home_account_payment" name="Overdue invoices" customize_show="True" inherit_id="portal.portal_my_home" priority="20">
+        <xpath expr="//div[hasclass('o_portal_docs')]" position="before">
+            <t t-set="portal_client_category_enable" t-value="True"/>
+            <t t-set="portal_alert_category_enable" t-value="True"/>
+        </xpath>
+        <div id="portal_alert_category" position="inside">
+            <t t-call="portal.portal_docs_entry">
+                <t t-set="bg_color" t-value="'alert alert-primary align-items-center'"/>
+                <t t-set="placeholder_count" t-value="'overdue_invoice_count'"/>
+                <t t-set="title">Invoices to pay</t>
+                <t t-set="url" t-value="'/my/invoices?filterby=overdue_invoices'"/>
+                <t t-set="show_count" t-value="True"/>
+                <t t-set="html_children">
+                    <button onclick="event.preventDefault(); window.location.href='/my/invoices/overdue'" class="btn btn-secondary ms-auto">
+                        Pay Now
+                    </button>
+                </t>
+            </t>
+        </div>
+    </template> 
```

Comment:
Made to display the pay now button in the "3 invoices to pay" widget. also explains the html_children enabling beforehand.

I need to check if that's the good way of doing it.